sandbox-backend/src/main/resources/template/mail.ftl

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
```

Comment:
Please add the result screen to trello issue as well as example.

Reply:
done